### PR TITLE
Homework 7 JDI completed

### PR DESCRIPTION
**Pull request by RinaDubinina**

### CODE REVIEW

File: src/main/java/hw_jdi/forms/MetalsAndColorsForm.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import hw_jdi.entities.MetalsAndColors;
+
+
+public class MetalsAndColorsForm extends Form {
```
Comment (troe88) [inline src/main/java/hw_jdi/forms/MetalsAndColorsForm.java]: This form should be parametrised by Entity class

File: src/main/java/hw_jdi/forms/MetalsAndColorsForm.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    @JDropdown(root = "#salad-dropdown",
+            value = ".dropdown-menu",
+            list = "li",
+            expand = ".caret")
+    private Dropdown vegetables;
+
+    @Css("#submit-button")
+    private Button submit;
+
+    public void setWebElements(MetalsAndColors metalsAndColors) {
```
Comment (troe88) [inline src/main/java/hw_jdi/forms/MetalsAndColorsForm.java]: What is the reason of this new method ? Why don't you override existing one ?

File: src/test/java/hw_jdi/dataProviders/DataPath.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package hw_jdi.dataProviders;
+
+public enum DataPath {
+
+    DATA_PATH("src/test/resources/hw_jdi/JDI_ex8_metalsColorsDataSet.json");
```
Comment (troe88) [inline src/test/java/hw_jdi/dataProviders/DataPath.java:5]: I don't see this file in your repository.

**Review by troe88 — CHANGES_REQUESTED** [review]

File: src/main/java/hw_jdi/forms/MetalsAndColorsForm.java (hunk 2 above)
Comment (RinaDubinina) [reply src/main/java/hw_jdi/forms/MetalsAndColorsForm.java]: I used @Override fill() method, and @Override pressButton() for Submit button.

File: src/main/java/hw_jdi/forms/MetalsAndColorsForm.java (hunk 1 above)
Comment (RinaDubinina) [reply src/main/java/hw_jdi/forms/MetalsAndColorsForm.java]: I  parametrised this form by MetalsAndColors class

File: src/main/java/hw_jdi/forms/MetalsAndColorsForm.java (hunk 2 above)
Comment (troe88) [reply src/main/java/hw_jdi/forms/MetalsAndColorsForm.java]: Okay, but why don't you override only one submit(...) method rather than those two ?

**Review by troe88 — APPROVED** [review]

**Pull request merged**